File: src/main/java/com/rkroom/blog/entity/Tags.java
package com.rkroom.blog.entity;

import javax.persistence.*;
import javax.validation.constraints.NotBlank;

@Entity

public class Tags {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;
    @NotBlank
    @Column(nullable = false)
    private String tag;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getTag() {
        return tag;
    }

    public void setTag(String tag) {
        this.tag = tag;
    }
}

File: src/main/java/com/rkroom/blog/entity/Tags.kt
package com.rkroom.blog.entity

import javax.persistence.*
import javax.validation.constraints.NotBlank

@Entity
data class Tags(@Id
                @GeneratedValue(strategy = GenerationType.IDENTITY)
                var id: Int? = null,
                @Column(nullable = false)
                var tag: @NotBlank String? = null
                )